modules/UI/UI.js

```diff
@@ -144,35 +144,34 @@ UI.toggleFullScreen = function() {
  * Notify user that server has shut down.
  */
 UI.notifyGracefulShutdown = function() {
-    messageHandler.openMessageDialog(
-        'dialog.serviceUnavailable',
-        'dialog.gracefulShutdown'
-    );
+    messageHandler.showError({
+        descriptionKey: 'dialog.gracefulShutdown',
+        titleKey: 'dialog.serviceUnavailable'
+    });
 };
 
 /**
  * Notify user that reservation error happened.
  */
 UI.notifyReservationError = function(code, msg) {
-    const message
-        = APP.translation.generateTranslationHTML(
-            'dialog.reservationErrorMsg',
-            {
-                code,
-                msg
-            });
-
-    messageHandler.openDialog(
-        'dialog.reservationError', message, true, {}, () => false);
+    messageHandler.showError({
+        descriptionArguments: {
+            code,
+            msg },
```

Comment:
style nit: maybe the curly should be on the next line? This was probably part of my changes.